app/views/users/circuitverse/edit.html.erb

```diff
@@ -67,6 +67,13 @@
   });
 </script>
 <script>
+
+  function remove()
+  {
+    $("#imgedit").attr("src", "/assets/thumb/Default-51ecc8a52348f20e7db5b8519e0535de1495413cd3c000b55bbd33a87f275231.jpg");
```

Comment:
This image path will change in each deployment

Reply:
Any other way with which I can achieve that, can you suggest me some way.